src/NuGet.Core/NuGet.Packaging/Signing/Utility/SignatureUtility.cs

```diff
@@ -256,21 +256,24 @@ public static IX509CertificateChain GetCertificateChain(PrimarySignature primary
                 includeChain);
         }
 
-        public static bool HasRepositoryCountersignature(PrimarySignature primarySignature)
+        public static bool HasRepositoryCountersignature(Signature signature)
```

Comment:
I think the changes on this line should be undone.  This method only applies to a primary signature.  It _could_ be used to test if a countersignature exists elsewhere, but there's no use case for that.